### ISSUE TITLE
Search Filter widget + Redirection

### ISSUE THREAD

**Issue by Robska-dev** [CrocoBlock/suggestions#2833]
After extensive contact with the support of Crcocoblock we came to the conclusion that the following feauture is missing and should be an essential part of the Crocoblock plugin, specifically the Search Filter widget.

I have looked around and could not find this in the current issues/suggestions.

I am looking for a way to search on the homepage of a website and be redirected to a specific results page. The Search Filter widget does not have this functionality.
I was told by the support that I should use the Apply Button widget to redirect me to the right page. The thing is that the Apply Button widget does not have a text search function attached to it.
What I am basically asking is to integrate the redirect function of the Apply Button widget with the Search Filter widget. That way users can search text using the Search Filter widget on one page and be redirected to a another specific results page.

This is a very important fucntionality as I can imagine lots of websites could use this for things like Job Offers and Products.
I have attached a website that has this functionality. [https://www.reed.co.uk/](url)
This example website uses two fields on the homepage, when you input text and click the search button or ENTER it redirects you to their Jobs page with the right results. That is precisely what I want and I am pretty sure a lot of your users would like to see that functionality.

This is a very important functionality for me and I would be really happy if the team can provide this to us.
Feel free to ask any questions.

**Comment by brupao** [comment]
+1

**Comment by Floydtm** [comment]
Let's discuss this question again. You must understand that WordPress has a default search results page. And all search queries will be redirected to this page. I can suggest you to make a custom search results page using jetthemecore plugin. You can place whatever you want on it. Let me know if this helps you. Thank you!

**Comment by erobertsme** [comment]
I agree that this is a pretty basic functionality that should be included in the JetSmartFilters plugin without needing to leverage another separate plugin or custom code to accomplish.
